packages/dialtone-tokens/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dialpad/dialtone-tokens",
-  "version": "1.23.5",
+  "version": "1.23.6-alpha.1",
```

Comment:
I think we don't want to merge this version to staging right?

Reply:
Right, I'll revert that changes.

I don't think we'll have issues with versions as the tags will be kept.